classes/class-wc-connect-shipping-label.php

```diff
@@ -362,6 +364,14 @@ protected function get_states_map() {
 		}
 
 		public function should_show_meta_box() {
+			if ( null === $this->show_metabox ) {
```

Comment:
Why did this separate class property (`$this->show_metabox`) and new function `should_show_meta_box()` get added? From what I can see, a new instance of `WC_Connect_Shipping_Label` is created each time it's called anyway, so seems like it's not needed.